### DISCUSSION TITLE
Show & Tell

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#14] [General help]
Have you made something you're proud of and want to show it off? Use the `Show and Tell` category to share your project. 

A couple of rules on Show & Tell:
1. No selling your product in our forum.
2. Be kind in your comments.
3. Use labels to designate whether you're sharing a `DG Project` or a `Personal Project`.
![DG Project](https://user-images.githubusercontent.com/34313413/188931520-1206538f-e3f7-4cfe-8c7d-9ba64c7a9927.png) ![Personal Project](https://user-images.githubusercontent.com/34313413/188931419-e6daa963-2e39-4777-93f3-530950317b98.png)

**Comment by bsixto** [comment]
![](https://user-images.githubusercontent.com/34313413/188931520-1206538f-e3f7-4cfe-8c7d-9ba64c7a9927.png)

Hello !

**Voxibot** integrates the **DeepGram** API to transcribe real-time phone calls on the **Asterisk** platform.
The real-time text can be written in a local text file, or pushed to a web server with HTTP json requests.

It allows having the complete transcription of a phone call at the hangup or being able to react to phone calls with a text analysis (NLP).

**Reply by BekahHW** [reply]
This is super cool! Thanks for sharing [Voxibot](https://voxibot.com/). I love the idea of text analysis!
